### ISSUE TITLE
snapshot teammanual

### ISSUE THREAD

**Issue by mvr320** [DOMjudge/domjudge#993]
### Description of the problem
When downloading https://www.domjudge.org/snapshot/domjudge-team-manual.pdf the first part of the summary looks strange.

### Your environment
Snapshot dir on domjudge.org

### Steps to reproduce
Download the pdf, open it.

### Expected behaviour
First part of summary is left aligned.

### Actual behaviour
First sentence is spread over the whole line

### Any other information that you want to share?
![image](https://user-images.githubusercontent.com/14887731/109424789-8b82aa00-79e5-11eb-9b57-5d61ef44b481.png)

**Comment by eldering** [comment]
I don't think this is a bug per sé. The whole text is justified to fill full lines. In this case, the URL cannot be broken (it seems) and causes poor spacing. I wouldn't want to change the text to right aligned just for this. Not sure there is a way to specify soft breakpoints like you can do in LaTeX.

**Comment by mvr320** [comment]
I understand your point (and if this is accepted its fine by me), a fix would be to restructure the text (to hide the problem) or use a footnote (https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes).

**Comment by eldering** [comment]
If a small change can fix this, that's fine with me, but I wouldn't want to make too weird/big changes just to fix a small layout issue.
